src/main/java/com/miliancode/demo/DemoApplication.java

```diff
@@ -1,9 +1,13 @@
-package com.example.demo;
+package com.miliancode.demo;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
+@EntityScan(basePackages = {"com.miliancode"})
+@ComponentScan(basePackages = {"com.miliancode"})
 public class DemoApplication {
```

Comment:
Move this class out of demo folder to root one and necessity of these annotation should disappear